### PR TITLE
fix: Generate correct source-map when content is not padded

### PR DESCRIPTION
**Pull request by znck**
Revert `source-map` to v0.6.1 as starting `0.7.0` there is no sync API.

### CODE REVIEW

File: lib/parse.ts [hunk 1]
```diff
@@ -108,5 +119,5 @@ function generateSourceMap(
       })
     }
   })
-  return map.toJSON()
+  return JSON.parse(map.toString())
```
Comment (sodatea) [inline lib/parse.ts]: Forgot to ask, why is this needed?

Comment (znck) [reply lib/parse.ts]: It's not required with `0.6.1`. Please revert this line.

Comment (sodatea) [reply lib/parse.ts]: 😂I think I found the answer…
There was a`.d.ts` file added to the `source-map` package since v0.5.7, but missing `toJSON` declaration in `SourceMapGenerator` until v0.7.0. So we can't use this method unless we patch its type definitions…
